Doc/library/functions.rst

```diff
@@ -111,15 +111,15 @@ are always available.  They are listed here in alphabetical order.
           return False
 
 
-.. function:: ascii(object)
+.. function:: ascii(obj, /)
```

Comment:
I'd leave the param name untouched and instead consider updating the docstring.

```suggestion
.. function:: ascii(object, /)
```

Reply:
Sorry, what docstring?

Reply:
https://github.com/python/cpython/blob/564385612cdf72c2fa8e629a68225fb2cd3b3d99/Python/bltinmodule.c#L400-L403

Perhaps "signature" is more accurate. Anyway, I'm not sure it is worth it (neither this PR nor updating the signature).

Reply:
Ah ok. I am not sure that it is necessary to edit many docstrings here: they are not wrong, as the params do not really have names.

See my comment on the ticket for a broader point about his issue.

Reply:
IMO, we should pick the clearer name and use that consistently. In this case, while using the full name rather than abbreviating is potentially a bit clearer on the _word_ intended, the latter avoids confusion on the _meaning_: `object` is already a builtin name, the base type object itself, while `obj` seems to be de-facto convention for function parameter that is an arbitrary object _instance_ to act on.